index.html

```diff
@@ -9,88 +9,103 @@
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
 </head>
-
 <body>
+	<div class="container">
```

Comment:
great start to the skeleton and good commit message. keep up the good code style 👍